server/db/fill-fake-data.js

```diff
@@ -166,6 +166,12 @@ function createFakeDuelWithAPI(config, usernames, MAX_ANSWER, index) {
 
       try {
         const [user1, user2] = faker.random.arrayElements(usernames, 2);
+        if (user1 === user2) {
+          console.warn(`     ${user1} can't defy himself/herself!`);
```

Comment:
him/her great great great... And what is the person is a non-binary person?

Reply:
Should be `himself/herself/himherself/herhimself/itself/whateverelse`

Reply:
If prefer `((him)?(her)?self|themselves)`